docs/dev/analysis/features/cht-axis-title.rst

```diff
@@ -0,0 +1,231 @@
+
+Axis Title
+==========
+
+A chart axis has a title which contains a text_frame that you can write
+text to. However, before you can add a title, you must set has_title
+to True.
+
+
+MS API Protocol
+---------------
+
+::
+
+    >>> chart = ActiveDocument.InlineShapes(1).Chart
+    >>> category_axis = chart.Axes(XlAxisType.xlCategory)
+    >>> category_axis.HasTitle
+    False
+    >>> category_axis.HasTitle = True
+    >>> category_axis.HasTitle
+    True
+    >>> category_axis.AxisTitle
+    <AxisTitle object ...>
+    >>> category_axis.AxisTitle.Text
+    ''
+    >>> category_axis.AxisTitle.Text = "Category Axis Title"
+    >>> category_axis.AxisTitle.Text
+    'Category Axis Title'
+
+
+    >>> chart = ActiveDocument.InlineShapes(1).Chart
+    >>> value_axis = chart.Axes(XlAxisType.xlValue)
+    >>> value_axis.HasTitle
+    False
+    >>> value_axis.HasTitle = True
+    >>> value_axis.HasTitle
+    True
+    >>> value_axis.AxisTitle
+    <AxisTitle object ...>
+    >>> value_axis.AxisTitle.Text
+    ''
+    >>> value_axis.AxisTitle.Text = "Value Axis Title"
+    >>> value_axis.AxisTitle.Text
+    'Value Axis Title'
+
+
+Python Usage
+------------
+
+::
+
+    >>> chart.category_axis.has_title
+    False
+    >>> chart.category_axis.title.text_frame.text = 'Category Axis Title'
+    ...
+    AttributeError: 'NoneType' object has no attribute 'text_frame'
+    >>> chart.category_axis.has_title = True
+    >>> chart.category_axis.title.text_frame.text = 'Category Axis Title'
```

Comment:
~~I'd be inclined to think we wouldn't necessarily want Title to *have* a text frame, but rather to *be* a text frame, like have all the needed methods and properties to manipulate the text.~~ Okay, now I see why it needs a text frame (c:rich is a CT_TextBody). It actually makes the inheritance easier too. So this adds some complexity because c:tx can alternately contain a c:rich or a c:strRef. So maybe we need to also have Title.has_text_frame and make it read/write; assigning True to Title.has_text_frame would remove any c:strRef that might be there and put in a minimal c:rich instead. Hmm, not sure what behavior assigning False to has_text_frame would have :) A little more noodling required there I expect.

I'm inclined to believe we need three analysis pages, or maybe two slightly different ones. Here's what I'm thinking. Title is really its own thing; and it's pretty complex all on its own. So I think it deserves a page of its own. Then these two pages can just describe down to getting a Title object reference. Since that mechanism would be so similar between the two, maybe those could be combined into a single page, but I wouldn't force it if it doesn't come naturally.

Title is going to have a lot of bits to it. It has the rich/strRef distinction, it can have a border I believe, it certainly has rich text (although that's just wiring up to TextFrame I expect), and I'm pretty sure it has color, like background color (I'm not sure if that can be wired into something completely already existing or not, but certain partially).

Not all that has to be implemented, of course, you just take it as far as you want. But in the analysis we have to see a path to how we get there so we don't create a dead end and later need to take an API change.